server/events/project_command_builder.go

```diff
@@ -151,13 +153,27 @@ func (p *DefaultProjectCommandBuilder) buildPlanAllCommands(ctx *CommandContext,
 	if err != nil {
 		return nil, err
 	}
+
 	// Parse config file if it exists.
 	hasRepoCfg, err := p.ParserValidator.HasRepoCfg(repoDir)
 	if err != nil {
 		return nil, errors.Wrapf(err, "looking for %s file in %q", yaml.AtlantisYAMLFilename, repoDir)
 	}
 
+	manifests, err := p.ManifestParser.FindManifestRoots(repoDir)
+	if err != nil {
+		return nil, errors.Wrapf(err, "looking for %s files in %q", yaml.ManifestYAMLFilename, repoDir)
+	}
+
+	hasManifestCfgs := len(manifests) > 0
+
 	var projCtxs []models.ProjectCommandContext
+
+	// As an initial step support for manifest files are only considered if
+	// there is no atlantis.yaml file in the root folder. Adding support for
+	// both of them at the same time is a little bit tricky, due to different
+	// edge cases where it will be possible to accidentaly create 2 different
+	// tf states for the same root.
```

Comment:
Assuming we'll want the same functionality when we extract this functionality to a binary?

Reply:
Do you mean supporting manifest.yaml + atlantis.yaml at the same time? I'm actually not sure if we should even support this in the future. It will just make it hard to maintain for unknown upside.